R/reorder.R

```diff
@@ -108,10 +108,10 @@ fct_inorder <- function(f, ordered = NA) {
 
 #' @export
 #' @rdname fct_inorder
-fct_infreq <- function(f, ordered = NA) {
+fct_infreq <- function(f, ordered = NA, .desc = TRUE) {
```

Comment:
This should be called `desc` I think, and can you please document it?

Reply:
Hmmm although then you'd call it `fct_infreq(x, desc = FALSE)` and the meaning doesn't seem that clear to me?